### PR TITLE
Add "Event" class

### LINKED ISSUE #59: Add "Event" class
The "Event" class:
- Is a separate idea from the current “tags” (tags are just a name)
- Can store a “from” and “to” date
- Can store an event description

### PR DESCRIPTION
**Pull request by limyixiang**
Fixes #59

### CODE REVIEW

File: src/main/java/seedu/address/model/event/UniqueEventList.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    }
+
+    public void setEvents(UniqueEventList replacement) {
+        requireNonNull(replacement);
+        internalList.setAll(replacement.internalList);
+    }
+
+    /**
+     * Replaces the contents of this list with {@code events}.
+     * {@code events} must not contain duplicate persons.
```
Comment (Nicholaskoh1) [inline src/main/java/seedu/address/model/event/UniqueEventList.java:89]: I think this is supposed to be duplicate events and not duplicate persons?

File: src/main/java/seedu/address/model/event/Event.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+import java.time.LocalDate;
+
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Represents an Event in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Event {
```
Comment (Nicholaskoh1) [inline src/main/java/seedu/address/model/event/Event.java:13]: LGTM!

**Review by Nicholaskoh1 — COMMENTED** [review]
LGTM! Found one possible typo in the code.

File: src/main/java/seedu/address/model/event/UniqueEventList.java (hunk 1 above)
Comment (limyixiang) [reply src/main/java/seedu/address/model/event/UniqueEventList.java:89]: Yes, nice catch, will fix that.

**Pull request merged**